tools/run-dev.py

```diff
@@ -159,9 +159,13 @@
 django_port = base_port + 1
 tornado_port = base_port + 2
 webpack_port = base_port + 3
+thumbor_port = base_port + 4
 
 os.chdir(os.path.join(os.path.dirname(__file__), '..'))
 
+# Thumbor server address
+os.environ['THUMBOR_HOST'] = '127.0.0.1:{0}'.format(thumbor_port)
```

Comment:
Does this environment variable do anything?  I don't see where it does, so it might be dead code.